### PR TITLE
Update date2 time

### PR DESCRIPTION
**Pull request by kduncklee**

### CODE REVIEW

**Comment by ctaylor451** [comment]
more comments

**Comment by andyl** [comment]
Please make a change.  Not accepted.

**Comment by andyl** [comment]
I'm closing this.

**Comment by ctaylor451** [comment]
reopened

File: date2.txt [hunk 1]
```diff
@@ -1 +1 @@
-Tue Jan  9 17:18:57 PST 2018
+Tue Jan  9 17:43:33 PST 2018
```
Comment (andyl) [inline date2.txt]: what the heck is this ???

**Review by andyl — CHANGES_REQUESTED** [review]
Please  make changes.

**Review by ctaylor451 — CHANGES_REQUESTED** [review]
pls make the following changes

**Review by andyl — APPROVED** [review]

File: date2.txt [hunk 2]
```diff
@@ -1 +1 @@
-Tue Jan  9 17:43:33 PST 2018
+Tue Jan  9 17:51:00 PST 2018
```
Comment (andyl) [inline date2.txt]: Maybe this should be 52

**Review by ctaylor451 — APPROVED** [review]

**Pull request merged**